src/containers/SearchPage/SearchPage.duck.js

```diff
@@ -131,15 +132,42 @@ export const searchListings = searchParams => (dispatch, getState, sdk) => {
       : {};
   };
 
-  const { perPage, price, dates, ...rest } = searchParams;
+  const availabilityParams = (datesParam, minDurationParam) => {
+    const dateValues = datesParam ? datesParam.split(',') : [];
+    const hasDateValues = datesParam && dateValues.length === 2;
+    const startDate = hasDateValues ? dateValues[0] : null;
+    const endDate = hasDateValues ? dateValues[1] : null;
+    const minDurationMaybe =
+      minDurationParam && Number.isInteger(minDurationParam) && hasDateValues
+        ? { minDuration: minDurationParam }
+        : {};
+
+    return hasDateValues
+      ? {
+          start: formatDateStringToUTC(startDate),
```

Comment:
This doesn't work. UTC is not used in hourly at all. Or is the API now unexpectedly expecting UTC in here? 

More questions: what kind of strings are used in the URL params? Do they contain time zone?

Anyway, FTW-hourly uses listing's time zone always, when presenting dates - and Flex transaction process doesn't care about time zones with time-based process just timestamps (except in availabilityPlan where listing's time zone is saved so that availability of "Tue 09:00 - 17:00" is understood correctly.)

So, If the listing is in NY and customer is in Helsinki, you need to adjust the timestamp so that start (provided by customer's browser) is converted to the same time-of-day in NY's time zone (which is a different timestamp).

So, the question is now that should that be done before the time is visible in the URL params or here. My gut feeling is that these URL's will be shared and therefore URL params should already reflect the listing's time zone or the time zone info should be completely absent.

Reply:
I don't quite understand how the listing time zone plays here. The user is making a query for listings, so there can be various time zones in the result.

What time zone should the query be in then? Not sure. Are our options either UTC or the user's time zone? Or is there a configured time zone that is "global" to the marketplace?

Reply:
@kpuputti 

If the Yoga session is held in Bali (i.e. customer tries to find yoga class for her vacation), then her assumption is that "when I'm on Bali I want to have yoga class at 10 am."

However, since that aforementioned customer is not at the Bali at the time of reservation but somewhere else, the Date object she gets from a browser is not pointing to correct time-of-day in Bali.

Let's say the customer is from Helsinki.
```
> local10am = new Date('2020-04-09 10:00:00')
// Thu Apr 09 2020 10:00:00 GMT+0300 (Eastern European Summer Time)
```

Use util/dates.js (`timeOfDayFromLocalToTimeZone`) to convert the time-of-day between time zones:
```shell
> listingTZ10am = timeOfDayFromLocalToTimeZone(local10am, 'Asia/Makassar')
// Browser would print that timestamp as:
// Thu Apr 09 2020 05:00:00 GMT+0300 (Eastern European Summer Time)
// because the 10 am in Bali is the same moment as 05 am in Helsinki.
```

If you want to render the timestamp correctly in listing's time zone in UI, there's util/dates helper function for that too, but for the sake of this example, I'll use Web Intl API directly:
```
> options = {
    year: 'numeric', month: 'numeric', day: 'numeric',
    hour: 'numeric', minute: 'numeric', second: 'numeric', 
    timeZone: 'Asia/Makassar',
    timeZoneName: 'short'
  };
> console.log(new Intl.DateTimeFormat('fi-FI', options).format(listingTZ10am));
// 9.4.2020 klo 10.00.00 UTC+8
```